graylog2-web-interface/src/views/stores/SearchJobs.ts

```diff
@@ -28,8 +28,10 @@ const startJobUrl = (id: string) => URLUtils.qualifyUrl(`/views/search/${id}/exe
 const cancelJobUrl = (nodeId: string, jobId: string) =>
   URLUtils.qualifyUrl(`/views/searchjobs/${nodeId}/${jobId}/cancel`);
 
-const pollJobUrl = (nodeId: string, jobId: string) =>
-  URLUtils.qualifyUrl(`/views/searchjobs/${nodeId}/${jobId}/status`);
+const pollJobUrl = (nodeId: string, jobId: string, page?: number, perPage?: number) =>
+  URLUtils.qualifyUrl(
+    `/views/searchjobs/${nodeId}/${jobId}/status${page && perPage ? `?page=${page}&per_page=${perPage}` : ''}`,
+  );
```

Comment:
I would like to create a helper function which just takes an object and generates the query string. We already have this logic in the `PaginatedURL` file, but I would like to make this change in a follow-up PR.